api/client/start.go

```diff
@@ -112,7 +112,7 @@ func (cli *DockerCli) CmdStart(args ...string) error {
 		})
 
 		// 3. Start the container.
-		if err := cli.client.ContainerStart(context.Background(), container); err != nil {
+		if err := cli.client.ContainerStart(context.Background(), container, ""); err != nil {
```

Comment:
Probably not a written best practice on context because it's available elsewhere, but yeah we nedd to fix the current code (mainly in `/api/client`. I'll work on it today :wink: